pupil_src/shared_modules/calibration_routines/finish_calibration.py

```diff
@@ -512,6 +473,40 @@ def select_calibration_method(g_pool, pupil_list, ref_list):
             )
 
 
+def dump_calibration_data(binocular, eye0, eye1, result):
```

Comment:
This function was probably used for testing purposes and should not be part of this PR. Even if it should stay, I would recommend to use `g_pool.user_dir` instead of a hard-coded path.

Reply:
Totally! I was not aware that when I push a commit, that it would go to "Needs review" automatically.  I thought, I would need to trigger that manually. And I pushed something which was not completely finished.